src/loader.mjs

```diff
@@ -26,8 +28,17 @@ const createLoader = () => {
       filePath => extname(filePath) === '.md'
     );
 
+    const progressBar = createProgressBar('Loading files');
+    progressBar.start(resolvedFiles.length, 0);
+
     return resolvedFiles.map(async filePath => {
       const fileContents = await readFile(filePath, 'utf-8');
+      progressBar.increment();
+      // normally we stop the progress bar when the loop is done
+      // but here we return the loop so we need to stop it when the last file is loaded
+      if (progressBar.value === progressBar.total) {
```

Comment:
Is the .stop() call needed? what happens if we simply stop incrementing when value === total?

Reply:
sometimes it's let the node process running for any reasons. So yeah I think it's needed to call `.stop()` method.
But it's can change if we use clack